borders.ftd

```diff
@@ -1,11 +1,10 @@
 -- record border-data:
-integer none:
+optional integer none:
```

Comment:
Why?

Reply:
This is not needed, you added it because you made optional and still provided a default value.

Reply:
@amitu This is because the `none` value is overwriting the uniform value.